charts/traction/README.md

```diff
@@ -1,6 +1,6 @@
 # Traction
 
-![version: 0.3.4](https://img.shields.io/badge/Version-0.3.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.5.1](https://img.shields.io/badge/AppVersion-0.5.1-informational?style=flat-square)
+![version: 0.3.4](https://img.shields.io/badge/Version-0.3.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square)
```

Comment:
Should be version 0.5.1 for the badge?

Reply:
Or no, that's chart version right?

Reply:
The app version is `1.0.0` - the chart version is `0.3.4` and was bumped after the prior release.